github-actions/verify-pr-title/src/verifyFeaturePrTitle.ts

```diff
@@ -0,0 +1,33 @@
+import { SEPARATOR, WORK_ITEM_PORTION } from './common';
+
+/**
+ * This regex portion matches the accepted options for a Feature Branch PR type.
+ * NOTE: Only one type may be selected at a time.
+ */
+const PR_TYPE_PORTION = '(NEW|FIX|CHANGE)';
+
+/**
+ * This regex portion matches any string wrapped in parentheses without additional
+ * parentheses inside it.
+ */
+const SCOPE_PORTION = '\\([^()]+\\)';
```

Comment:
We may or may not even need this for the vs code extension. The scope will probably always just be VSCode Extension. :-)